domino_generator/example/ex3/web-project/components/named-div/colorbox.html

```diff
@@ -1,10 +1,20 @@
 <red-box>
-    <div style="background-color: red; width: 100px; height: 100px">
+    <d-style>
```

Comment:
This is good, and we should keep it, but it should also support reading similar content from `colorbox.scss`. IDEs are better to support CSS rules that.